Src/Support/IntegrationTests/OidcTests.cs

```diff
@@ -44,6 +45,10 @@ public async Task OidcTokenProvider_ServiceAccountCredential()
 
             var verificationOptions = new SignedTokenVerificationOptions();
             verificationOptions.TrustedAudiences.Add("https://this.is.a.test");
+            // The test is flaky without this, because on acassion everything happens so fast
```

Comment:
acassion => occasion
(Assuming that's what you mean?)

Reply:
Done, will push shortly and merge on green.